onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -239,17 +239,45 @@ def aten_conv_depthwise3d(
     raise NotImplementedError()
 
 
+@torch_op("aten::cross_entropy_loss", trace_only=True)
```

Comment:
Is trace_only required here?

Reply:
Yes. because 
1) there has one bug in onnx-script  when parse Optional argument.
2) this function contains no op() in logic, so have to use trace_only.